### DISCUSSION TITLE
Support vhost selection

### DISCUSSION THREAD

**Discussion by E1v1s** [rabbitmq/rabbitmq-perf-test#530] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

Hey dear community, i'm stuck in a situation where i created quorum queue under new vhost and try make observability of cluster with perftest app.
Unfortunately i got error `reply-text=NOT_ALLOWED - access to vhost '/' refused for user` and it seems to be correct because i'm cannot change vhost in perftest in uri or other settings.
Please correct me if i wrong, is it possible to point to correct, non-default vhost? 

### Describe the solution you'd like

another flag with vhost selection

**Comment by michaelklishin** [comment, marked as answer]
You can specify any `--uri` you want and with it, a virtual host. See https://www.rabbitmq.com/uri-spec.html.
